docs/articles/posts/20241129_microsoft_hack.md

```diff
@@ -0,0 +1,52 @@
+---
+title: NHS x Microsoft AI Hackathon
+authors: [AmaiaImazBlanco]
+date: 2024-12-04
+categories: 
+    - NHS Websites
+    - LLMs
+    - Python
+    - Events
+links:
+    - TBC
+slug: microsoft-hackathon
+description: >
+    The NHS England Data Science team, as well as a range of other analysts from across the organisation, attended an AI Hackathon at Microsoft, organised by the Data Science Team together with Microsoft and Kainos, with the key stakeholders being the NHS Websites Team. 
+---
+
+> The NHS England Data Science team, as well as a range of other analysts from across the organisation, attended an AI Hackathon at Microsoft, organised by the Data Science Team together with Microsoft and Kainos, with the key stakeholders being the NHS Websites Team. In this article, the author shares her experiences at the event. 
+
+<!-- more -->
+
+Months ago, when I was invited to this hackathon, I sat at my desk and reluctantly signed up because everyone else was attending, sighing to myself about "please not another hackathon", and this was the energy I brought with me to the beautiful Storey Club, in Paddington, London, last week Tuesday. I thought "well at least I'll get a free lunch", but I was proven so wrong in my pessimism. Every team had either a Kainos or Microsoft representative, as well as a range of participants from across NHS England, including at least a few Data Science Team members per team. And unlike a usual hackathon, this one had stakeholders and well defined use cases, we were all working to one common goal. 
```

Comment:
Maybe make this intro more upbeat! I think we want to highlight this a positive and I dont think there is much to be gained with faux pessimism. Maybe you were fired up for another hacakthan, how lucky you are to be able to attend them frequently!

Reply:
rather than saying "a few data science team members pert team" maybe highlight that there was a strong NHS England Data Science team presence, with several Data Science team members on each team.